### PR TITLE
maxlength overhaul

### PR DESCRIPTION
**Pull request by adamdruppe**

### CODE REVIEW

File: app/views/enrollments/_form.html.erb [hunk 1]
```diff
@@ -51,15 +51,15 @@
                 <h4>Your Name</h4>
                 <div class="form-group has-success">
                   <%= f.label :first_name %>
-                  <%= f.text_field :first_name, :class => 'form-control short', :required => 'required' %>
+                  <%= f.text_field :first_name, :class => 'form-control short', :required => 'required', :maxlength => '35' %>
                 </div>
                 <div class="form-group has-success">
                   <%= f.label :middle_name %>
-                  <%= f.text_field :middle_name, :class => 'form-control short' %>
+                  <%= f.text_field :middle_name, :class => 'form-control short', :maxlength => '70' %>
```
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: Actually, the last name falls in the same category as middle name re: length in some cultures.  Not a big deal though.

File: app/views/enrollments/_form.html.erb [hunk 2]
```diff
@@ -51,15 +51,15 @@
...
                 </div>
                 <div class="form-group has-success">
                   <%= f.label :last_name %>
-                  <%= f.text_field :last_name, :class => 'form-control short', :required => 'required' %>
+                  <%= f.text_field :last_name, :class => 'form-control short', :required => 'required', :maxlength => '35' %>
```
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: Actually, the last name falls in the same category as middle name re: length in some cultures. Not a big deal though.

File: app/views/enrollments/_form.html.erb [hunk 3]
```diff
@@ -210,11 +210,11 @@
                 <h5>What is your current occupation?:</h5>
                 <p class="help">Please respond to at least one of the following questions about your occupation (industry, company, title).</p>
                 <label>Industry:</label>
-                  <%= f.text_field :industry, "class"=>'form-control' %><br />
+                  <%= f.text_field :industry, "class"=>'form-control', 'maxlength' => '75' %><br />
```
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: This is 40 chars on Contact.Industry.

Comment (adamdruppe) [inline app/views/enrollments/_form.html.erb]: On Wed, Apr 15, 2015 at 11:40:05AM -0700, Brian Sadler wrote:
> This is 40 chars on Contact.Industry.

This one is CampaignMember.Industry though, do they map to the same?

I think I'll make the length of contact.industry larger though.

File: app/views/users/_volunteer.html.erb [hunk 4]
```diff
@@ -13,7 +13,7 @@
       Profession / job title:
     </div>
     <div class="col-sm-7">
-      <%= f.text_field :profession, :class => 'form-control' %>
+      <%= f.text_field :profession, :class => 'form-control', :maxlength => '80' %>
```
Comment (sadleb) [inline app/views/users/_volunteer.html.erb]: You set it to 70 for employers but 80 for volunteers?

File: app/views/enrollments/_form.html.erb (hunk 3 above)
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: > This one is CampaignMember.Industry though, do they map to the same?
> 
> I think I'll make the length of contact.industry larger though.

Yeah, they map to the same (well they are copied).  I see that you updated both to 80 in salesforce which is the right thing to do.

File: app/views/enrollments/_form.html.erb (hunk 2 above)
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: > Actually, the last name falls in the same category as middle name re: length in some cultures. Can we make this 70 too. The Salesforce max length is 80.

I'm just waiting on this line before I merge.

Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: > I'm just waiting on this line before I merge.

I'll just merge and change it on github so I can start testing.

**Pull request merged**